tools/etl/tg-jdbc-driver/README.md

```diff
@@ -8,18 +8,52 @@ The TigerGraph JDBC Driver is a Type 4 driver, converting JDBC calls directly in
 | --- | --- | --- | --- | --- | --- |
 | 1.0 | 2.2.4+ | 1.8 | Rest++ | JSON | Support builtin, compiled queries and loading jobs |
 | 1.1 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support tabular format and Spark |
-| 1.2 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
+| 1.2.x | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
```

Comment:
There're just v1.2.1 and v1.2.2 on maven.com, shall we remove entries for v1.0 and v1.1 and add v1.2.1?